test/infrastructure/kind/mapper.go

```diff
@@ -79,6 +79,62 @@ type Mapping struct {
 var preBuiltMappings = []Mapping{
 
 	// TODO: Add pre-built images for newer Kind versions on top
+	// Pre-built images for Kind v1.22.
```

Comment:
```suggestion
	// Pre-built images for Kind v0.22.
```

(also for the other existing comments :-) there is no kind v1